### PR TITLE
Create a new handbook page for curating the editor experience

### PR DESCRIPTION
**Pull request by annezazu**
## What?

This proposes a v1 of a new page that compiles various methods for curating the editor experience. Thanks to @justintadlock @carolinan @bph for chiming in/reviewing! In the future, I'd love to see a section with specific recommendations for Agencies and Enterprise level folks. My hope is that an initial version can get out into the open and folks in those spaces can contribute back with use studies, feedback on what would be helpful, etc. 

## Why?

Right now, the information around how best to curate the experience/tailor it to your needs is spread across multiple different resources. This seeks to pull them into one place and explain how they each can be used. 

## Please help with this PR!

I still struggle at times to get things in place for new docs so, if you're game to help, it would be lovely for folks to contribute to this PR both to move it forward and suggest changes.

### CODE REVIEW

**Comment by annezazu** [comment]
Noting that I definitely added the visuals incorrectly (didn't see the assets folder). I'll try to figure this out now :)

**Comment by annezazu** [comment]
I can't seem to figure out why the video won't properly render. When I look at the individual file, it seems the size is still too big to display. If I don't add it to the assets file and just drop it in, it works though. If anyone can help here, that would be rad :)

File: docs/how-to-guides/curating-the-editor-experience.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+}
+```
+
+## Limiting interface options with theme.json
+
+**Limit options on a per block basis**
+
+Beyond defining default values, using theme.json allows you to also remove options entirely and instead rely on what the theme has set in place. Below is a visual showing two extremes with the same paragraph block: 
+
+![Image of restricted interface](https://github.com/WordPress/gutenberg/blob/annezazu-curate-new-doc/docs/assets/Locking%20comparison%20visual.png)
```
Comment (fabiankaegy) [inline docs/how-to-guides/curating-the-editor-experience.md]: I think the URL of the image already needs to point to the path of the image once it has been merged to trunk.

**Review by fabiankaegy — COMMENTED** [review]
I really like this guide. It is a great overview of the various options that one has :)

File: docs/how-to-guides/curating-the-editor-experience.md (hunk 1 above)
Comment (annezazu) [reply docs/how-to-guides/curating-the-editor-experience.md]: Ah! Okay. Let me update that :) I just was worried they wouldn't appear if I reference them in trunk without them being there first. Makes sense though.

Comment (annezazu) [reply docs/how-to-guides/curating-the-editor-experience.md]: Updated but, as you can see now, the images don't show up. I guess that's what has to be done but I'm not sure what folks have done previously.

**Comment by gziolo** [comment]
You need to run `npm run docs:build` to regenerate the manifest file used at https://developer.wordpress.org/block-editor/.

**Comment by ryanwelcher** [comment]
> You need to run `npm run docs:build` to regenerate the manifest file used at https://developer.wordpress.org/block-editor/.

@annezazu I took the liberty of rebuilding the docs and pushing the changes up.

**Review by ryanwelcher — APPROVED** [review]
LGTM! 👍

**Pull request merged**